Installer.cls

```diff
@@ -48,6 +45,13 @@ ClassMethod setup(ByRef pVars, pLogLevel As %Integer, pInstaller As %Installer.I
   #; Let our XGL document generate code for this method. 
   Do %code.WriteLine(" If $Data(pVars, path)#10,path'="""" Set pVars(""MODDIR"")=path")
   Do %code.WriteLine(" Set pVars(""CURNS"")=$namespace")
+  Do %code.WriteLine(" Set prevNS = $namespace")
+  Do %code.WriteLine("  New $Namespace")
+  Do %code.WriteLine("  Set $Namespace = ""%SYS"" ")
+  Do %code.WriteLine("  kill properties")
+  Do %code.WriteLine("  Set sc = $ClassMethod(""Config.Namespaces"", ""Get"", prevNS, .properties)")
```

Comment:
really, what's wrong with this way?
`##class(Config.Namespaces).Get(prevNS, .properties)`

Reply:
I agree. @Beiming-Zhang In generated code, can always use direct ##class